DEVELOPMENT.md

```diff
@@ -211,6 +211,8 @@ TTN_LW_IS_EMAIL_NETWORK_IDENTITY_SERVER_URL="http://localhost:8080/oauth.js"
 TTN_LW_CONSOLE_UI_ASSETS_BASE_URL="http://localhost:8080/assets"
 ```
 
+> Note: Development configuration can be loaded from environment variables only. It cannot be sourced from a config file.
+
```

Comment:
This is in the wrong place. The configuration above can very well be source from the config file. This line needs to be at L217 and say something like 
> Note: webpack-related configuration can be loaded from environment variables only. It cannot be sourced from a config file.